examples/nas/multi-trial/mnist/search.py

```diff
@@ -40,21 +49,76 @@ def forward(self, x):
         x = F.max_pool2d(self.conv2(x), 2)
         x = torch.flatten(self.dropout1(x), 1)
         x = self.fc2(self.dropout2(F.relu(self.fc1(x))))
-        output = F.log_softmax(x, dim=1)
```

Comment:
Then where is the softmax?

Reply:
It's already in `CrossEntropyLoss`.

Reply:
OK, so the change is due to the migration from lightning to custom training loop.
Why do we abandon lightning? Seems CGO only supports lightning trainer.

Reply:
We didn't abandon lightning.
I only updated the beginner example here so that users don't have to learn lightning first before using NAS. I hope this change will smoothen the learning curve.

For advanced features like CGO, lightning is still recommended (actually required).